lib/daru/io/importers/excel.rb

```diff
@@ -45,22 +55,26 @@ class Excel < Base
         #   #    3        4    Franz      nil    Paris      nil
         #   #    4        5   George      5.5     Tome    a,b,c
         #   #    5        6  Fernand      nil      nil      nil
-        def initialize(path, worksheet_id: 0)
+        def initialize(path, sheet: 0, headers: true)
```

Comment:
For the excelx (xlsx) importer, name of the keyword argument has been chosen as `sheet`, as it could even be a `String` and `worksheet_id` seems a bit misleading to only `Integer` type (Of course, the documentation should take care of it). And to maintain consistency between excel and excelx importer, `worksheet_id` was renamed to `sheet`.

But this was prior to commit f9a8356, where excel importer redirected to excelx importer and both methods were forced to have same argument names. Now that redirect is done by `register_io_module`, this can certainly be reverted back to `worksheet_id`.